DTS-STN/canadian-dental-care-plan
# Application Route Placeholder Added

### Description
A new route with minimal content has been added as a placeholder for the later additional content.

### Related Azure Boards Work Items


AB#2931

### Screenshots (if applicable)


![Screenshot from 2024-02-19 15-57-28](https://github.com/DTS-STN/canadian-dental-care-plan/assets/154452024/97807266-8b70-4065-8d9e-c3961c3428d2)

### Checklist


### Test Instructions


1.     Run the application.
2.     Navigate to the index page of the application.
3.     Select the new application link
4.     Observe the new application page
